lib/arena.rb

```diff
@@ -1,2 +1,77 @@
+# require_relative "gladiator"
+# require "pry"
+
 class Arena
+  attr_accessor :name, :gladiators
+
+  def initialize name
+    @name = name.capitalize
+    @gladiators = []
+  end
+
+  def add_gladiator gladiator
+    if @gladiators.length < 2
+      @gladiators << gladiator
+      @gladiators.sort_by! do |gladiator|
+        gladiator.weapon
+      end
+      @gladiators
+    else
+      puts "There are 2 gladiators in the arena already."
+    end
+  end
+
+  def remove_gladiator gladiator
+      @gladiators.delete_if {|i| i.name == gladiator}
+      @gladiators
+  end
+
+  def entertained?
+    @gladiators.each do |gladiator|
+      if gladiator.name.index("Maximus") != nil
+        puts "The crowd is entertained."
+        return true
+      elsif @gladiators[0].name != "Maximus" && @gladiators[1].name != "Maximus"
+        puts "The crowd is NOT entertained."
+        return false
+        break
+      end
+    end
+  end
+
+  def fight
+    if @gladiators.length <= 1
+      puts "The arena needs more gladiators!"
+    else
+      @gladiators.each do |gladiator|
+        if gladiator.name == "Maximus"
+          puts "Maximus is always the winner."
+          @gladiators.delete_if {|gladiator| gladiator.name != "Maximus"}
+        elsif @gladiators[0].weapon == "Spear" && @gladiators[1].weapon == "Trident"
+          @gladiators.shift
+          puts @gladiators[0].name + " is the winner!"
```

Comment:
This method is a little long and more difficult to read. Could you think of a way to condense this into fewer lines of code?